### ISSUE TITLE
Problem with ElvUi minimap panel after last update.

### ISSUE THREAD

**Issue by Zakakanec** [mooreatv/NeatMinimap#14]
Hey, after last update of ElvUi NeatMinimap also hides time panel on my minimap, how could I fix it? Thanks for your future answer.

**Comment by stornquist** [comment]
Go to neatminimap.lua and find `NMM.exlude`

replace the whole  thing with
```
NMM.exclude = {
  -- Our own frame
  ["NeatMinimapFrame"] = true,
  -- Std blizz stuff
  ["MiniMapMailFrame"] = true,
  ["MiniMapBattlefieldFrame"] = true,
  ["MinimapBorder"] = true,
  ["MinimapBackdrop"] = true,
  ["MiniMapInstanceDifficulty"] = true, -- bfa
  ["GuildInstanceDifficulty"] = true,
  ["MiniMapChallengeMode"] = true,
  ["GameTimeFrame"] = true, -- handled through config, ElvUI (or bfa?) reparents it
  -- SexyMap
  ["SexyMapCustomBackdrop"] = true,
  ["SexyMapPingFrame"] = true,
  -- ElvUI
  ["MMHolder"] = true,
  ["HelpOpenWebTicketButton"] = true,
  ["HelpOpenTicketButton"] = true,
  ["TopMiniPanel"] = true,
  ["LeftMiniPanel"] = true,
  ["RightMiniPanel"] = true,
  ["BottomMiniPanel"] = true,
  ["TopLeftMiniPanel"] = true,
  ["TopRightMiniPanel"] = true,
  ["BottomLeftMiniPanel"] = true,
  ["BottomRightMiniPanel"] = true,
  -- Questie 4.1 onward
  ["QuestieFrameGroup"] = true,
  -- ElvUI
  ["MinimapPanel"] = true
}
```

This will add the datatext panel to the ignore list. I added a request to merge this into the addon as well.

**Comment by mooreatv** [comment]
fixed in #15 will be in next release
